frontends/election-manager/src/components/prose.tsx

```diff
@@ -1,89 +0,0 @@
-import styled from 'styled-components';
-
-interface Props {
-  compact?: boolean;
-  density?: number;
-  maxWidth?: boolean;
-  textCenter?: boolean;
-}
-
-export const Prose = styled('div')<Props>`
-  margin: ${({ textCenter }) => (textCenter ? 'auto' : undefined)};
-
-  /* width: 100%; */
-  max-width: ${({ maxWidth = true }) => (maxWidth ? '66ch' : undefined)};
-  text-align: ${({ textCenter }) => (textCenter ? 'center' : undefined)};
-  line-height: ${({ density }) => (density !== 0 ? '1.1' : '1.3')};
```

Comment:
This PR does have one "breaking" change: the default line-height in VxAdmin UI is changing from `1.3` to `1.2`.

This was never intended to be 1.3 in the UI of the app, only in the printed HMPB. The value of `1.3` is preserved for the HMPB.